src/testing/factories/juju/juju.ts

```diff
@@ -152,7 +153,17 @@ export const modelDataFactory = Factory.define<ModelData>(() => ({
   "remote-applications": {},
 }));
 
+export const crossModelQueryStateFactory = Factory.define<CrossModelQueryState>(
+  () => ({
+    results: null,
+    errors: null,
```

Comment:
These are not necessary in every place they appear, but I thought that the tests become more descriptive if we include them.

Reply:
Yep, this is good as it is.